apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -370,6 +372,22 @@ export default class ScriptEditor extends React.Component {
                 </p>
               </HelpTip>
             </label>
+            <label>
+              Can be translated (only check this if the script is stable)
+              <input
+                name="is_translateable"
+                type="checkbox"
+                defaultChecked={this.props.isTranslateable}
+                style={styles.checkbox}
+              />
+              <HelpTip>
+                <p>
+                  If checked, this unit will be included in our translation
+                  pipeline. Major changes to this unit after translation has
+                  started can break translations.
```

Comment:
@bethanyaconnor it might be worthwhile to leave the `ScriptConstants.i18n?` method in place in the short term, and just update it to reference this new logic. We can then do a pass through the codebase to update references to that method and remove it as a separate PR

Reply:
That makes sense! This PR doesn't touch that method (I left that as a follow-up) but I can just modify that function instead of removing it.

Reply:
@bethanyaconnor I just merged the PR that adds a couple calls to i18n? . Here it is for easy reference: https://github.com/code-dot-org/code-dot-org/pull/36219